app/Models/Location.php

```diff
@@ -253,6 +253,21 @@ public function assignedAssets()
         return $this->morphMany(\App\Models\Asset::class, 'assigned', 'assigned_type', 'assigned_to')->withTrashed();
     }
 
+    public function assignedAccessories()
+    {
+        return $this->morphToMany(
+            Accessory::class,
+            'accessory_assignment',
+            'accessories_checkout',
+            'assigned_to',
+            'assigned_type',
+            null,
+            null,
+            'accessory_assignment',
+            true
+        );
```

Comment:
Just glanced at this and immediately felt the pain of getting the arguments of `morphToMany` right...